index.html

```diff
@@ -0,0 +1,131 @@
+            </div>
+            <div class="program-2">
+              <img src="images/program_icon_02.svg" alt="program-icon" class="program-img">
+              <h4 class="program-text">Ehibition</h4>
+              <p class="program-overview-text">Appreciate various creations applying Microsoft license of developers, organized Microsoft</p>
+            </div>
+            <div class="program-3">
+              <img src="images/program_icon_03.svg" alt="program-icon" class="program-img">
+              <h4 class="program-text">Forum</h4>
+              <p class="program-overview-text">Have the time to share your thoughts and opinions with experts for each topic.</p>
+            </div>
+            <div class="program-4">
+              <img src="images/program_icon_04.svg" alt="program-icon" class="program-img">
+              <h4 class="program-text">Workshop</h4>
+              <p class="program-overview-text">Try creating your own work using open source license rather than just watching at it.</p>
+            </div>
+            <div class="program-5">
+              <img src="images/program_icon_05.svg" alt="program-icon" class="program-img">
+              <h4 class="program-text">Ignite</h4>
+              <p class="program-overview-text">get opportunities to network with Microsoft affiliates around the world, also after the summit.</p>
+            </div>
+          </div>
+          <a href="#" class="program-link">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <!--section 3 built dynamically-->
+      <section class="section-3">
+        <div class="sec-3-head">
+          <h4 class="sec-3-header">Featured speakers</h4>
+          <div class="sec-3-line"></div>
+        </div>
+        <div class="sec-3-container">
+
+        </div>
+      </section>
+      <!--section 3 built dynamically-->
+      <section class="section-4">
+        <div class="patner-header">
+          <h4 class="patner-title">Patner</h4>
+          <div class="seperate-line-2"></div>
+        </div>
+        <div class="sponsors">
+          <img src="images/dam.svg" alt="company-logos" class="company-logos">
+          <img src="images/mozilla 1.svg" alt="company-logos" class="company-logos">
+          <img src="images/air.svg" alt="company-logos" class="company-logos">
+          <img src="images/naver 1.svg" alt="company-logos" class="company-logos">
+          <img src="images/gog.svg" alt="company-logos" class="company-logos">
+        </div>
+      </section>
+      <section class="section-5">
+        <div class="license">
+          <img src="images/microsoft.png" alt="logo-icon" class="footer-image">
+          <div class="license-text">
+            <p class="text-1">
+              2023 Developer's summit. Some Rights Reserved.
+            </p>
+          </div>
+        </div>
+      </section>
+    </main>
```

Comment:
On mobile, the page ends with a `see more` button and not a footer😉. also, there are just a couple of speakers and not all the speakers😉. let's make this change to make your project match the deisgn.😉

design
<img width="328" alt="image" src="https://github.com/2004-okari/CAPSTONE-01/assets/64643995/18679175-efa8-4eca-adf1-0c3503b2142d">

yours
<img width="331" alt="image" src="https://github.com/2004-okari/CAPSTONE-01/assets/64643995/8437a89f-02e8-4b85-9ef8-a05503f213e6">

also take note of the spacings around the speaker cards and implement the changes as necessary👍.